api4/webhook_test.go

```diff
@@ -61,6 +63,38 @@ func TestCreateIncomingWebhook(t *testing.T) {
 	_, resp = Client.CreateIncomingWebhook(hook)
 	CheckNoError(t, resp)
 
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
+		hook.UserId = th.BasicUser2.Id
+		defer func() { hook.UserId = "" }()
+
+		newHook, response := client.CreateIncomingWebhook(hook)
+		CheckNoError(t, response)
+		require.Equal(t, th.BasicUser2.Id, newHook.UserId)
```

Comment:
Would be great to check for the `CreatorID` too

Reply:
Incoming webhook uses `UserId` and outgoing webhook uses `CreatorId`, so each test needs to check for a different one :sweat_smile:

Reply:
Oh, I thought it was the same. GH folds everything and I got confused. Nvm